examples/metadata.go

```diff
@@ -25,6 +25,16 @@ func MetadataExample() {
 		Location: "https://identity-provider.com/saml/post/logout",
 	})
 
+	idp.AddContactPerson(saml.MetadataBinding{
```

Comment:
@sccalabr AddContactPerson function should add ContactPersion not MetadataBinding